docs/reference/alert-schema.asciidoc

```diff
@@ -0,0 +1,120 @@
+[[alert-schema]]
+
+== Alert schema
+
+{elastic-sec} stores alerts that have been generated by detection rules in hidden {es} indices.  In 7.x versions, the index pattern was `.siem-signals-<spacename>`.  In 8.x versions, the index pattern is `.alerts-security.alerts-<spacename>`, and the field names have changed. The following table includes the new names and provides a cross-reference to the previous field names.
+
+NOTE: Users are advised NOT to use the `_source field` in alert documents, but rather use the fields API to programmatically obtain the list of fields used in these documents. Learn more about {ref}/search-fields.html[retrieving selected fields from a search].
```

Comment:
This implies that there is something called "the fields API," which I don't think is quite right. This is more consistent with the info on the linked page:
```suggestion
NOTE: Users are advised NOT to use the `_source` field in alert documents, but rather to use the `fields` option on the search API to programmatically obtain the list of fields used in these documents. Learn more about {ref}/search-fields.html[retrieving selected fields from a search].
```